### DISCUSSION TITLE
new installed autogenstudio failed at startup during database creation

### DISCUSSION THREAD

**Discussion by Jeffhaha** [microsoft/autogen#4306] [Q&A]
Hi all, 
as I start autogenstudio as official tutorial :
autogenstudio ui --port 8081
2024-11-22 08:36:50.538 | INFO     | autogenstudio.utils.utils:get_db_uri:253 - Using database URI: sqlite:////Users/jinglu/.autogenstudio/database.sqlite
2024-11-22 08:36:50.538 | INFO     | autogenstudio.utils.utils:init_app_folders:288 - Initialized application data folder: /Users/jinglu/.autogenstudio
INFO:     Started server process [76174]
INFO:     Waiting for application startup.
***** App started *****
2024-11-22 08:36:50.916 | INFO     | autogenstudio.database.utils:init_db_samples:150 - Initializing database with Default and Travel Planning Workflows
**2024-11-22 08:36:50.917 | INFO     | autogenstudio.database.dbmanager:create_db_and_tables:49 - Error while initializing database samples: 'validated_data' must be provided if 'call_default_factory' is True.**
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:8081 (Press CTRL+C to quit)

I meet the  error "autogenstudio.database.dbmanager:create_db_and_tables:49 - Error while initializing database samples: 'validated_data' must be provided if 'call_default_factory' is True.**", I can find nowhere to edit the parameter that it mentioned. Does anyone meet the same problem?

I install and run autogenstudio within a conda v-env with python 3.11. 
in the browser though I can open webpage, but I can use no function there, I met always error " 422 Unprocessable Entity
".
Thanks a lot for anyhelp. 

Jinglu

**Comment by rysweet** [comment]
@victordibia

**Comment by victordibia** [comment]
Hi @Jeffhaha 

Can you let me know what version of autogenstudio you have?  
This line in your error `autogenstudio.database.dbmanager:create_db_and_tables` suggests that you have an older version of autogenstudio. 

```bash
pip show autogenstudio
```

The latest version is a dev build. - [0.4.0.dev37](https://pypi.org/project/autogenstudio/0.4.0.dev37/).
Please see the main autogenstudio discussion [here also](https://github.com/microsoft/autogen/discussions/4208)

```bash
pip install autogenstudio==0.4.0.dev37
```

Please try that and let me know if you are still seeing issues.
